### ISSUE TITLE
warning with icc compiler

### ISSUE THREAD

**Issue by jaustermann** [geodynamics/aspect#1589]
I get the following warning in all files that use SphericalShell. This is with the compiler version:
icc version 15.0.0 (gcc version 4.4.7 compatibility)
```
In file included from /n/home09/jaustermann/aspect_new/aspect/source/initial_temperature/spherical_shell.cc(23):
/n/home09/jaustermann/aspect_new/aspect/include/aspect/geometry_model/spherical_shell.h(48): warning #411: class template "aspect::GeometryModel::SphericalShell<dim>" defines no constructor to initialize the following:
In file included from /n/home09/jaustermann/aspect_new/aspect/source/initial_temperature/spherical_shell.cc(23):
            const member "aspect::GeometryModel::SphericalShell<dim>::spherical_manifold"
In file included from /n/home09/jaustermann/aspect_new/aspect/source/initial_temperature/spherical_shell.cc(23):
            const member "aspect::GeometryModel::SphericalShell<dim>::boundary_shell"
In file included from /n/home09/jaustermann/aspect_new/aspect/source/initial_temperature/spherical_shell.cc(23):
            const member "aspect::GeometryModel::SphericalShell<dim>::straight_boundary"
In file included from /n/home09/jaustermann/aspect_new/aspect/source/initial_temperature/spherical_shell.cc(23):
      class SphericalShell : public Interface<dim>, public SimulatorAccess<dim>
```

**Comment by bangerth** [comment]
This would make for an easy starter project :-)

**Comment by hlokavarapu** [comment]
This PR is ready to be tested.

**Comment by gassmoeller** [comment]
Closed by #1643.
